pontoon/base/static/js/fluent_interface.js

```diff
@@ -293,7 +293,7 @@ var Pontoon = (function (my) {
           $('#ftl-area .attributes ul:first').append(attributes);
 
           // Update access keys presentation
-          $('#ftl-area .attributes textarea').keyup();
+          $('#ftl-area textarea:visible').keyup();
```

Comment:
this feels like quite a loose selector - `#ftl-area textarea.value:visible` maybe ?